core/frontend/src/main/scala/io/udash/bindings/Bindings.scala

```diff
@@ -395,6 +396,36 @@ trait Bindings {
   implicit def toPropertyOps[T](property: ReadableProperty[T]): PropertyOps[T] =
     new PropertyOps(property)
 
+  implicit class InlineStyleOps[T: HasCssName](style: T) {
+    /** Sets style on element. */
+    def applyTo(element: HTMLElement, value: String = ""): Unit =
+      element.style.setProperty(implicitly[HasCssName[T]].cssName(style), value)
+
+
+    /** Removes style from element. */
+    def removeFrom(element: HTMLElement): Unit =
+      element.style.setProperty(implicitly[HasCssName[T]].cssName(style), "")
+
+    /** Synchronises style value with property content. */
+    def bind(property: ReadableProperty[String]): Binding =
+      property.reactiveApply {
+        case (elem: HTMLElement, null) => removeFrom(elem)
+        case (elem: HTMLElement, v) => applyTo(elem, v)
+        case _ =>
+      }
+
+    /**
+      * Synchronises style value with property content by adding it when property is not null and
+      * condition property is 'true' and removing otherwise.
+      */
+    def bindIf(property: ReadableProperty[String], conditionProperty: ReadableProperty[Boolean]): Binding =
+      property.combine(conditionProperty)((_, _)).reactiveApply {
```

Comment:
As you will see in the proposed tests extension this binding does not clear the listeners properly. There are two possible solutions: 
* comment out the listeners count checks and report an issue to fix the `combine` method,
* fix the `combine` method (it should not register listeners in the origin properties as long as there is no listener in the combined property itself - take a look at the implementation of `transform` method).

Reply:
#195